clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapServerStore.java

```diff
@@ -221,12 +221,16 @@ boolean handleOversizeMappingException(long hash) {
   }
 
   public void close() {
+    boolean success = false;
     writeLockAll();
     try {
       clear();
-      segments.clear();
+      success = true;
```

Comment:
What does this buys us? There is no _expected_ failure path once we managed to `writeLockAll()` so we should simply move the `segments.clear()` after the finally block.